src/mailchimp/mailchimp_manager.py

```diff
@@ -8,6 +8,13 @@
     * Implement methods
     * Implement datalogging
 """
+import json
+
+import base64
+
+import mailchimp_marketing as MailchimpMarketing
+
+from mailchimp_marketing.api_client import ApiClientError
 
 from typing import *
```

Comment:
The `typing` module is built-in to Python, meaning it should be above the Mailchimp 3rd party libraries.